lib/resolve-repositories.js

```diff
@@ -79,6 +79,6 @@ export async function resolveRepositories(state, repositories) {
 
   process.stdout.write("\n");
 
-  // return array with unique repositories based by id (https://stackoverflow.com/a/56757215)
-  return resolvedRepositories.filter((repo, index, repoList) => repoList.findIndex(v2 => (v2.id === repo.id)) === index);
+  // return array with unique repositories based by name (https://stackoverflow.com/a/56757215)
+  return resolvedRepositories.filter((repo, index, repoList) => repoList.findIndex(v2 => (v2.name === repo.name)) === index);
```

Comment:
I think we should use `full_name` here to avoid situations where repo names are the same in different organisations. What do you think @gr2m

Reply:
I'd definitely prefer to compare by `id`

Reply:
Sure, no problem